aip/general.md

```diff
@@ -1,6 +1,7 @@
 ---
 aip_index:
-  scope: general
```

Comment:
because the "general" aips don't have a scope, it is easier to remove the scope from here instead of add it to all of them.